config/locales/en.yml

```diff
@@ -497,6 +497,9 @@ en:
               blank: "^Select a study mode"
             applications_open_from:
               blank: "^Select an applications open date"
+            accrediting_provider:
+              blank: "Select an accrediting provider"
+              is_not_accredited: "Update the accredited provider (it's no longer accredited)"
```

Comment:
deferring to @Kateharries & @MaeveRoseveare  on this one – had a look through [gov error messaging](https://design-system.service.gov.uk/components/error-message) do we need to slightly tweak the validation message?

Reply:
@JR-G @MaeveRoseveare @MylesJarvis I think the message should just say 'Update the accredited provider' and then we have content that appears on the 'Accredited providers' page just for this scenario (see my other comment).

Reply:
Me and @Kateharries have agreed to remove the bit in brackets. If we've missed the boat on this PR as it's been approved by James, I'll do it immediately after we ship this.

Reply:
We also discussed some improved copy changes we can make on this, that we will ticket up separately.